src/datasets.js

```diff
@@ -311,6 +311,7 @@ export default {
         if (options.skipHeaderRow) query.skipHeaderRow = 1;
         if (options.desc) query.desc = 1;
         if (options.bom) query.bom = 1;
+        else if (options.bom === false) query.bom = 0;
```

Comment:
This seems like a really dirty fix of something. Why do we need this special behavior for `bom`?

Reply:
I agree, it looks quite bad, but I think we need to have it there. A comment would be good.

Reply:
It's because default value of bom is `true`

Reply:
We should have called this variable `skipBom` to have it alligned

Reply:
Too late now. Actually, default value of `bom` is "depends on the format", for CSV it's true, for JSON it's not etc. Therefore `skipBom` wouldn't cut it either. Can we just add a comment explaining this?

Reply:
Comment updated